presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/credential/CredentialProviderModule.java

```diff
@@ -58,7 +58,7 @@ protected void setup(Binder binder)
     private void bindCredentialProviderModule(CredentialProviderType name, Module module)
     {
         install(installModuleIf(
-                BaseJdbcConfig.class,
+                BaseJdbcAuthenticationConfig.class,
```

Comment:
It's implicitly bound here. Can you please add explicit bind for the new config class?

Reply:
`BaseJdbcConfig` also does not have explicit bind.

Reply:
Ah it has as `configBinder(binder).bindConfig(BaseJdbcConfig.class);` in connector implementations